Consistent usage of "quickstart"

andynava opened the pull request.

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

andynava committed:
Consistent usage of "quickstart"

andynava committed:
Firewall section

andynava reviewed (commented):
Mention that firewall rules are opt-in (whitelisted)
Update sentence that talks about whitelisting a smaller IP range

andynava committed:
Restore data section

andynava commented on articles/postgresql/tutorial-design-database-using-azure-portal.md:
```diff
@@ -100,7 +100,7 @@ When we created our Azure Database for PostgreSQL server, the default **postgres
 ## Connect to PostgreSQL database using psql in Cloud Shell
 
-Let's now use the psql command-line utility to connect to the Azure Database for PostgreSQL server. 
+Let's now use the [psql](https://www.postgresql.org/docs/9.6/static/app-psql.html) command-line utility to connect to the Azure Database for PostgreSQL server. 
```
Since this is the first time the tool is being introduce, it should be linked

andynava committed:
Clear up a bit of confusion

In the original sentence, it read as if the rows are updated when data is retrieved.

andynava committed:
First step is creating a server
